core/admin/mailu/configuration.py

```diff
@@ -16,13 +16,14 @@
     'DOMAIN_REGISTRATION': False,
     'TEMPLATES_AUTO_RELOAD': True,
     'MEMORY_SESSIONS': False,
+    'FETCHMAIL_ENABLED': True,
```

Comment:
Do we really want to se this to true?
The expected default behaviour is that if fetchmail is not configured it is disabled. 

It does not have to be backwards compatible. When a new Mailu version is released, we always state that you must regenerate your docker-compose.yml & mailu.env file (which will contain the new env var).

Of course this will also be mentioned in the release notes.

More importantly, the fetchmail container itself will only run if the env var is explicitly set to true. So for people not updating their mailu.env file, they will still see fetchmail in the admin GUI, but will have a non working fetchmail container.

Reply:
Hmm. Initially I did set the default to False. Because it is an optional component, it seems reasonable to me that it should be enabled explicitly.  However @ghostwheel42 mentioned that it might be better if we don't break existing installations by defaulting to False.

> More importantly, the fetchmail container itself will only run if the env var is explicitly set to true. So for people not updating their mailu.env file, they will still see fetchmail in the admin GUI, but will have a non working fetchmail container.

In the current state, fetchmail is ran by default. It only skips the runs if `FETCHMAIL_ENABLED` is set to False. This is the same for enabling the UI for fetchmail.

I guess there is something to say for both defaults, but in my opinion defaulting to false and mentioning it in release notes seems the most logical.

Reply:
@ghostwheel42 
Do you agree with us? We always explicitly mention in the release notes that the docker-compose.yml file must be regenerated. When you do this, the mailu.env file will be correctly configured.

Of course the release notes will also mention this new setting for the people who `feel lucky` and rather manually update the docker-compose.yml file & mailu.env file upon upgrade.

Existing deployments are not affected. Only new deployments on the new mailu release. If people don't use setup.mailu.io and also do not read the release notes, then it is not our problem.

Reply:
Yes, sure. If it’s documented it’s okay.